lib/core-integration/src/Test/Integration/Scenario/API/Network.hs

```diff
@@ -51,14 +53,34 @@ import qualified Network.HTTP.Types.Status as HTTP
 spec :: forall t. SpecWith (Context t)
 spec = do
     it "NETWORK - Can query network information" $ \ctx -> do
-        eventually_ $ do
+        r <- eventually $ do
             now <- liftIO getCurrentTime
             r <- request @ApiNetworkInformation ctx networkInfoEp Default Empty
             expectResponseCode @IO HTTP.status200 r
             verify r
                 [ expectFieldSatisfy nextEpoch ((> now) . epochStartTime)
                 , expectFieldEqual syncProgress Ready
                 ]
+            return r
+
+        let currentEpochNum =
+                getFromResponse (#networkTip . #epochNumber . #getApiT) r
+        let nextEpochNum =
+                getFromResponse (#nextEpoch . #epochNumber . #getApiT) r
+        nextEpochNum `shouldBe` currentEpochNum + 1
+
+    it "NETWORK - Calculated next epoch is the next epoch" $ \ctx -> do
+        r1 <- request @ApiNetworkInformation ctx networkInfoEp Default Empty
+        let calculatedNextEpoch = getFromResponse (#nextEpoch . #epochNumber) r1
+        let nextEpochStartTime = getFromResponse (#nextEpoch . #epochStartTime) r1
```

Comment:
Perhaps add the following here:
```hs
        now <- liftIO getCurrentTime
        nextEpochStartTime > now `shouldBe` True
```

Reply:
I think I haven't added it because it is checked in previous test case.